src/app/token/TokenPage.js

```diff
@@ -105,32 +102,30 @@ class TokenPage extends Component<Props, State> {
                     <div className='token-countdown-container'>
                       <Countdown small title='Time Left' deadline={token.expires} />
                     </div>
-                    <h2 className='pui-h2'>
-                      Create Your Security Token
-                    </h2>
+                    <h2 className='pui-h2'>Create Your Security Token</h2>
                     <h3 className='pui-h3'>
-                      Create your security token before your token reservation expires.
-                      If you let your token reservation expire, the token symbol you selected will be
-                      available for others to claim.
+                      Create your security token before your token reservation expires. If you let your token
+                      reservation expire, the token symbol you selected will be available for others to claim.
                     </h3>
                     <h3 className='pui-h3'>
-                      To proceed with the creation of your security token,
-                      we recommend you work with your Advisory to answer
-                      the following questions:
+                      To proceed with the creation of your security token, we recommend you work with your Advisory to
+                      answer the following questions:
                     </h3>
                     <br />
                     <CompleteTokenForm onSubmit={this.handleCompleteSubmit} />
                   </div>
                 </div>
-              ) : ''}
-              {token.address ? (
-                <MintTokens />
-              ) : ''}
+              ) : (
+                ''
```

Comment:
and here...
why do you add 2 more lines for this if we only want to show something in one case without else?

Reply:
I think it's simpler to read since it's harder to miss. Will revert.